frontend/src/features/member/MemberFilter.tsx

```diff
@@ -0,0 +1,152 @@
+import { css } from "@emotion/core";
+import styled from "@emotion/styled";
+import { Button, Popover, Box } from "@material-ui/core";
+import AssigneeAutoComplete from "components/AssigneeAutoComplete";
+import { fetchBoardById } from "features/board/BoardSlice";
+import { createErrorToast } from "features/toast/ToastSlice";
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { BoardMember } from "types";
+import { selectAllMembers } from "./MemberSlice";
+
+const FilterButton = styled.div`
+  margin-left: 0.5rem;
+`;
+
+const Content = styled.div`
+  padding: 2rem;
+`;
+
+const Description = styled.p`
+  margin-top: 0;
+  margin-bottom: 1rem;
+  font-size: 18px;
+  font-weight: bold;
+`;
+
+interface Props {
+  boardId: number;
+}
+
+const MemberFilter = ({ boardId }: Props) => {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const [filteredAssignee, setFilteredAssignee] = useState<BoardMember[]>([]);
+  const dispatch = useDispatch();
+  const members = useSelector(selectAllMembers);
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const postFilterMember = async (users: BoardMember[]) => {
+    try {
+      dispatch(
+        fetchBoardById({
+          boardId: boardId,
+          assigneeIds: users.map((m) => m.id),
+        })
+      );
+      handleClose();
+    } catch (err) {
```

Comment:
`fetchBoardById` already has a `catch` block for API errors and this is probably never reached. Perhaps move the toast to that `catch` block